http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/DbReference.scala

```diff
@@ -100,3 +105,7 @@ class DataAccess(val profile: JdbcProfile)(implicit val executionContext: Execut
     sql"select version()".as[String]
   }
 }
+
+final class DBIOOps[A](private val dbio: DBIO[A]) extends AnyVal {
+  def transaction(implicit dbRef: DbReference[IO]): IO[A] = dbRef.inTransaction(dbio)
```

Comment:
I wonder if this `DBIOOps` class should have another method that takes a `TransactionIsolation` param and passes it to `inTransaction`

Reply:
I don't see `TransactionIsolation` being used anywhere..but I can definitely add it